docs/source/configuring_zipkin.rst

```diff
@@ -108,6 +108,21 @@ fine tune as per your use case.
     The method MUST take `request` as a parameter (so that you can make trace
     id deterministic).
 
+
+5. zipkin.set_extra_binary_annotations
+--------------------------------------
+    A method that takes the `request` object as a parameter and produces
```

Comment:
It is worth mentioning that the return value of the callback must be a `dict` and each of the `key` and `value` __MUST__ be in `str` format.